core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
```

Comment:
Note that, regardless of where you put the CSS, it generally has global effect -- this is because all the templates get concatenated. The reason the CSS here might only show up in exploration_player.html is because that file only gets loaded on the exploration player page. Putting it in oppia.css wouldn't work.

That said, this is looking a bit messy -- in general I would expect just the only CSS classes here to be those for oppia-exploration-header. So, some design questions:

- On the rest of the site, should the 'brand name' icon be made smaller generally on mobile?
- On the rest of the site, should the changes made to the breadcrumbs / breadcrumb icons apply generally on mobile (to all pages)?